### PR TITLE
Stripe: Add application fee on EMV authorize calls

### PR DESCRIPTION
**Pull request by bizla**
Stripe's EMV API accepts application fee requests, to be requested on `authorize` requests instead of `capture`. So, this simply adds the application fee in an `authorize` when dealing with an EMV payment. 

@ryanbalsdon @girasquid for review.

### CODE REVIEW

**Comment by ryanbalsdon** [comment]
:grapes:

**Comment by bizla** [comment]
@aprofeit care to :eyes: ?

File: lib/active_merchant/billing/gateways/stripe.rb [hunk 1]
```diff
@@ -64,6 +64,7 @@ def authorize(money, payment, options = {})
           r.process do
             post = create_post_for_auth_or_purchase(money, payment, options)
             post[:capture] = "false" unless emv_payment?(payment)
```
Comment (girasquid) [inline lib/active_merchant/billing/gateways/stripe.rb]: The unless/if right next to each other points to this maybe being better as an if/else

**Comment by girasquid** [comment]
:+1:

**Comment by bizla** [comment]
Merging on green CI.

**Comment by aprofeit** [comment]
:+1:

**Pull request closed without merging**